specs/latest/1.0/webgl.idl

```diff
@@ -55,6 +61,7 @@ dictionary WebGLContextAttributes {
     WebGLPowerPreference powerPreference = "default";
     boolean failIfMajorPerformanceCaveat = false;
     boolean desynchronized = false;
+    WebGLPixelFormat pixelFormat = "implicit";
```

Comment:
nit: file needs regeneration

Reply:
+1, needs to be regenerated to pick up `undefined` as the default.